server/chroma/src/routes/v1/photo/create.rs

```diff
@@ -203,6 +209,31 @@ fn resize_and_save(
             }
         };
 
+        // Verify, this also puts it in the cache, nice speedup for later
+        trace!("Checking if uploaded image actually works on AWS");
```

Comment:
Succeeds currently when running with MinIO, however due to the problem in photo::photo_to_proto_url() this method can **never** succeed when running with MinIO.